crates/lib/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "liquid-lib"
+version = "0.19.1"
+authors = ["Johann Hofmann <mail@johann-hofmann.com>"]
+description = "The liquid templating language for Rust"
+repository = "https://github.com/cobalt-org/liquid-rust/tree/master/liquid-lib"
+documentation = "https://docs.rs/liquid-lib"
+readme = "README.md"
+categories = ["template-engine"]
+keywords = ["liquid", "template", "templating", "language", "html"]
+license = "MIT"
+edition = "2018"
+
+[badges]
+azure-devops = { project = "cobalt-org", pipeline = "liquid-rust" }
+
+[dependencies]
+liquid-error = { version = "^0.19", path = "../error" }
+liquid-derive = { version = "^0.19", path = "../derive" }
+liquid-compiler = { version = "^0.19", path = "../compiler" }
+liquid-interpreter = { version = "^0.19", path = "../interpreter" }
+liquid-value = { version = "^0.19", path = "../value" }
+liquid-core = { version = "^0.19", path = "../core" }
+kstring = { version = "0.1.0", path = "../kstring" }
+serde = { version = "1.0", optional = true, features = ["derive"] }
+itertools = "0.8.0"
+regex = "1.0"
+unicode-segmentation = "1.2"
+url = "1.5"
+chrono = "0.4"
+once_cell = "1.0"
+deunicode = { version = "1.0.0", optional = true }
+
+[dev-dependencies]
+serde_yaml = "0.8"
+
+[features]
+default = ["extra-filters", "serde"]
+extra-filters = []
+jekyll-filters = ["deunicode"]
```

Comment:
Let's set these up as
- `stdlib`
- `jekyll`
- `all`

with `stdlib` being the default